docs/docsite/rst/collections_tech_preview.rst

```diff
@@ -229,9 +229,9 @@ You can upload ollection artefacts with Mazer, as shown in the following example
 The above command triggers an import process, just as if the collection had been uploaded through the Galaxy website. Use the **My Imports**
 page to view the output from the import process.
 
-Your API key can be found on `the preferences page in Galaxy </me/preferences>`_.
+Your API key can be found on `the preferences page in Galaxy <https://galaxy.ansible.com/me/preferences>`_.
 
-To learn more about Mazer, see `Mazer <https://galaxy-dev.ansible.com/docs/mazer/index.html>`.
+To learn more about Mazer, see `Mazer <https://galaxy-dev.ansible.com/docs/mazer/index.html>`_.
```

Comment:
@samccann @bcoca should this be `galaxy-dev` or `galaxy`?

Reply:
dont think it matters much anymore, they used to have very diff docs, but now they just have minor style changes.